Makefile

```diff
@@ -471,18 +471,20 @@ docker-start-bash:
 docker-start-dev:
 	docker run --rm \
 	-v ${CURRENT_DIR}:/mnt/local_unstructued \
-	-ti ${DOCKER_IMAGE}
+	-ti --entrypoint /bin/bash ${DOCKER_IMAGE}
```

Comment:
@tabossert - This worked when I specified the entrypoint in `docker run`, but didn't work with `/bin/bash` as `CMD` in the `Dockerfile`. Any idea what would cause that?